src/Microsoft.Data.Sqlite/CacheMode.cs

```diff
@@ -3,10 +3,20 @@
 
 namespace Microsoft.Data.Sqlite
 {
+    /// <summary>
+    /// Represents the connection-caching modes that can be used when creating a new <see cref="SqliteConnection"/>. <seealso href="https://www.sqlite.org/sharedcache.html">See SQLite.org for complete documentation on the differences between caching modes.</seealso>
+    /// </summary>
     public enum CacheMode
     {
         // TODO consider adding a 'default' or 'unspecified' mode
+        /// <summary>
+        /// Default mode. Each connection uses a private cache.
+        /// </summary>
         Private,
+
+        /// <summary>
+        /// Shared-cache mode. Connection share a cache. This mode can change the behavior of transaction and table locking.
```

Comment:
Is this PR still alive? If so, change `Connection` to `Connections` in the second sentence.